diff --git a/node_modules/roslib/src/RosLib.js b/node_modules/roslib/src/RosLib.js
index 3f6768d..5246078 100644
--- a/node_modules/roslib/src/RosLib.js
+++ b/node_modules/roslib/src/RosLib.js
@@ -8,7 +8,7 @@
  *
  * If you use nodejs, this is the variable you get when you require('roslib').
  */
-var ROSLIB = this.ROSLIB || {
+var ROSLIB = {
   /**
    * @default
    * @description Library version
